lisa/analysis/rta.py

```diff
@@ -222,6 +222,15 @@ def get_log_file(self, task):
         """
         return self.perf_data[task]['logfile']
 
+    @memoized
+    def _start_time_from_bundle(self):
```

Comment:
same comment as for the previous commit, this can already be given as `get_df('foo', start=trace.start)`. This kind of default behavior tend to over interlink all classes with non-straightforward semantic, and lead to not exposing fundamental parameters in intermediate APIs, making refactoring a nightmare.